build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.12.2'
+        classpath 'com.android.tools.build:gradle:1.0.0'
```

Comment:
this is the android-gradle-plugin, which runs at 1.0.0. Gradle itself, which it depends on, runs at 2.2.1.